firmware/firmware.cpp

```diff
@@ -1083,6 +1087,12 @@ void FlashLoader::handle_data_end(bool success) {
         flash_mapped = false;
       }
       blit_enable_user_code();
+
+      if(blit_is_launcher_running()) {
```

Comment:
Only difference in my local code is a comment: `// soft-reset launcher if it's running to update list`